pytorch_lightning/loggers/tensorboard.py

```diff
@@ -152,7 +156,14 @@ def save(self) -> None:
         hparams_file = os.path.join(dir_path, self.NAME_HPARAMS_FILE)
 
         # save the metatags file
-        save_hparams_to_yaml(hparams_file, self.hparams)
+        if Container is not None:
+            if isinstance(self.hparams, Container):
+                from omegaconf import OmegaConf
+                OmegaConf.save(self.hparams, hparams_file)
```

Comment:
lgtm.
If we get user feedback about unresolved arguments we can add resolve=True here, but this is fine.